docs/api/webview-tag.md

```diff
@@ -847,6 +847,37 @@ Returns:
 Emitted when any frame (including main) starts navigating. `isInPlace` will be
 `true` for in-page navigations.
 
+### Event: 'will-redirect'
+
+Returns:
+
+* `url` String
+* `isInPlace` Boolean
+* `isMainFrame` Boolean
+* `frameProcessId` Integer
+* `frameRoutingId` Integer
+
+Emitted as a server side redirect occurs during navigation. For example a 302
+redirect.
+
+This event will be emitted after `did-start-navigation` and always before the
+`did-redirect-navigation` event for the same navigation.
+
+Calling `event.preventDefault()` does __NOT__ have any effect.
```

Comment:
Hm, what's the value in having two events here if `will-redirect` can't be canceled?

Reply:
good point :) I will remove the `will-redirect` one